spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureCloudConfigProperties.java

```diff
@@ -128,18 +128,24 @@ public void setMsi(AzureCloudConfigMSIProperties msi) {
 
     @PostConstruct
     public void validateAndInit() {
+        Assert.notEmpty(this.stores, "At least one config store has to be configured.");
+
         if (!msiEnabled) {
-            Assert.notEmpty(this.stores, "At least one config store has to be configured.");
+            int uniqueStoreSize = this.stores.stream().map(s -> s.getConnectionString()).distinct()
```

Comment:
Should we check config store name instead of connection string?  Connection string for the same config store may have different format?

Reply:
Check config store name only make sense, but still we do not support multi connection string for the same config store. Updated to check config store name only.